app/src/main/java/com/sdp/movemeet/utility/UserDatabase.java

```diff
@@ -0,0 +1,92 @@
+package com.sdp.movemeet.utility;
```

Comment:
You can add some documentation in this class

Reply:
Good point, I will work on it